backend/api/models/footprint_form.py

```diff
@@ -0,0 +1,19 @@
+import uuid
+
+from django.db import models
+
+from .mixins import SoftDeleteMixin
+
+
+class FootprintForm(SoftDeleteMixin):
+    id = models.UUIDField(
```

Comment:
Not sure which type of id is better, the auto-incremental Integer or UUID?
@tai271828 @yanghaochang104

Reply:
My gut feeling is auto-incremental integer, but UUID is good for me as well.